srcpkgs/glfw/template

```diff
@@ -1,19 +1,19 @@
 # Template file for 'glfw'
 # update together with glfw-wayland
 pkgname=glfw
-version=3.3.8
+version=3.4
 revision=1
 build_style=cmake
-configure_args="-DBUILD_SHARED_LIBS=ON"
+configure_args="-DBUILD_SHARED_LIBS=ON -DGLFW_BUILD_WAYLAND=OFF"
 makedepends="MesaLib-devel libXrandr-devel libXi-devel glu-devel libXcursor-devel
  libXinerama-devel"
-short_desc="Multi-platform library for creating windows with OpenGL contexts"
-maintainer="Enno Boland <gottox@voidlinux.org>"
+short_desc="Multi-platform library for OpenGL, OpenGL ES and Vulkan"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
why?

Reply:
Description taken from the GLFW project.
Email address not reachable. Orphaned email address is used in the glfw-wayland package or vice versa.

Reply:
glfw package hasn't been updated for almost two years. So I expect it to be abandoned.

Reply:
gottox isn't very active but is a member of the core team. don't orphan it

Reply:
Then I'll write him in both packages.

Can I reach him somehow?

Reply:
don't change the maintainer fields at all.

@Gottox

Reply:
It's fine to change it to orphaned for me.